src/main/resources/com/google/api/codegen/js/sample.snip

```diff
@@ -114,16 +114,37 @@
 
       @end
       function {@class.initClientFuncName}() {
+        @if class.auth.type != "NONE"
+          var API_KEY = '';  // TODO: Update placeholder with desired API key.
```

Comment:
I'm confused, or maybe I just forgot what we talked about way back: If auth is not NONE, then we necessarily have an API  key? What about 3LO, etc?